src/modules/auth/reducer.js

```diff
@@ -5,6 +5,7 @@ const initialState = {
 	resources: [],
 	unauthorizedNavItem: [],
 	unauthorizedNavChildren: [],
+	sessionExpiration: false
```

Comment:
@Aringoaway For sake of the action naming, I propose to rename it to `sessionExpired`